### ISSUE TITLE
disable trailingClosures as default?

### ISSUE THREAD

**Issue by Maxatma** [nicklockwood/SwiftFormat#394]
`trailingClosures` rule can break some code, when used on default:
```
self.collectionView.performBatchUpdates { [weak self] in
    //somecode
}
```
I guess coz of 
`    open func performBatchUpdates(_ updates: (() -> Void)?, completion: ((Bool) -> Void)? = nil) // allows multiple insert/delete/reload/move calls to be animated simultaneously. Nestable.
`

Similar to our discussion in https://github.com/nicklockwood/SwiftFormat/issues/361

**Comment by nicklockwood** [comment]
Hmm, I think this is probably best handled with an explicit black-list. But I'll think about it.

**Comment by nicklockwood** [comment]
@Maxatma I've fixed this particular case in version 0.40.4. Let me know if you find others.
